openedx/adg/lms/our_team/admin.py

```diff
@@ -0,0 +1,18 @@
+"""
+Registering models for our team app.
+"""
+from django.contrib import admin
+
+from .models import OurTeamMember
+
+
+@admin.register(OurTeamMember)
+class ApplicationHubAdmin(admin.ModelAdmin):
+    """
+    Django admin class for OurTeam
+    """
+
+    fields = ('name', 'designation', 'image', 'description', 'url',)
+    list_display = ('id', 'name', 'designation', 'image', 'description',)
+    list_filter = ('name',)
```

Comment:
I guess having a list filter by designation is more appropriate if needed. Filter by name is not feasible as it will add a filter for each name.